tasks/eu/geo.py

```diff
@@ -279,388 +279,6 @@ def populate(self):
-  ', Landkreis', ''),
-  ', Stadtkreis', ''),
-  ' (DE)', ''),
-  'Landkreis Rostock', 'Rostock'))
-  as nuts_name, name nuts_original,
-  nullif(name LIKE '%Stadt%', false) AS is_stadt
-  FROM "eurostat.geo".importnutsnames__99914b932b
-  where code ilike 'DE%'
-    and lower(name) not like '%, kreis'
-    and code not like '%\_%'
-    and code not like '__%Z%'
-    and name not like '%(NUTS%'
-    and length(code) = 5
-    and name not in ('Not regionalised / Unknown NUTS 3'
-                     , 'Baden', 'Württemberg',
-                     'Nordrhein-Westfalen, Rheinland-Pfalz and Saarland'
-                     )
-), nuts as (
-  select row_number() over (order by nuts_name, is_stadt) nuts_row, is_stadt,
-  --code,
-  nuts_name, nuts_original
-  from nuts_unprocessed
-  order by nuts_name, is_stadt
-), shn as (
-  select row_number() over (order by namn, ara) shn_row,
-  ara, ppl,
-  --shn,
-  namn shn_name
- from "eurostat.geo".importshnnames__99914b932b
- where icc ilike 'DE' and use = 4
- order by namn, ara
-) select nuts_original, shn_name, ara, ppl, is_stadt, nullif(
-nuts_name =
-lower(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(shn_name,
-    ' a.d. ', ' an der '),
-    ' i.d. ', ' in der '),
-    'OPf.', 'Oberpfalz'),
-    '(Oldb)', '(Oldenburg)'),
-    ' i. ', ' im '),
-    ' a. ', ' am '))
-, false)
-from nuts, shn where nuts_row = shn_row;
-
-
-select
-  icc,
-  count(distinct a.nama) shn_names,
-  count(distinct b.name) nuts3_names
-FROM "eurostat.geo".importshnnames__99914b932b a,
-   "eurostat.geo".importnutsnames__99914b932b b
-where length(code) = 5
-  AND code not like '%\_%'
-  AND name not like '%(NUTS%'
-  and code not like '__%Z%'
-  AND CASE WHEN lower(icc) = 'gb' THEN 'uk'
-           ELSE lower(icc)
-  END = lower(substr(code, 1, 2))
-  and lower(CASE
-    WHEN icc IN ('CZ') THEN REPLACE(name, ' kraj', '')
-    WHEN icc in ('DE') THEN REPLACE(REPLACE(REPLACE(REPLACE(name,
-    ', Kreisfreie Stadt', ''),
-    ', Landkreis', ''),
-    ' (DE)', ''),
-    'Landkreis Rostock', 'Rostock')
-    ELSE name
-  END) = lower(CASE
-    WHEN icc IN ('BE') THEN 'Arr. ' || namn
-    WHEN icc IN ('BG') THEN nama
-    WHEN icc IN ('DE') THEN REPLACE(REPLACE(REPLACE(namn,
-    ' a.d. ', ' an der '),
-    ' i.d. ', ' in der '),
-    ' a. ', ' am ')
-    ELSE namn
-  END)
-  AND use = CASE
-    WHEN icc IN ('LI', 'ME') THEN 1
-    WHEN icc IN ('BG', 'CH', 'CZ', 'HR', 'NO', 'RO', 'SE', 'SK') THEN 2
-    WHEN icc IN ('ES', 'FI', 'FR', 'HU', 'IT', 'PT') THEN 3
-    WHEN icc IN ('DE', 'BE') THEN 4
-  END
-group by icc
-order by icc;
-
-
-select icc, nama, namn
-FROM "eurostat.geo".importshnnames__99914b932b
-WHERE namn NOT IN (
-  select a.namn
-  FROM "eurostat.geo".importshnnames__99914b932b a,
-       "eurostat.geo".importnutsnames__99914b932b b
-  where a.namn ILIKE b.name
-  AND a.icc in (select distinct substr(b.code, 1, 2) from "eurostat.geo".importnutsnames__99914b932b)
-)
-;
-'''
+class AllNUTSGeometries(WrapperTask):
```

Comment:
`AllNUTSGeometries` could extend `MetaWrapper` instead of `WrapperTask` so it would be collected by `collect_meta_wrappers` and automatically tested